src/main/java/com/danigu/blog/comment/persistence/CommentEntity.java

```diff
@@ -0,0 +1,29 @@
+package com.danigu.blog.comment.persistence;
```

Comment:
I would consider CommentEntity as part of the domain model, not part of the persistence layer. If you are using ORM that maps domain objects into the database, your persistence layer would only map business objects into the db.